tests/test_users/test_schemas.py

```diff
@@ -9,13 +9,20 @@
 from src.apps.user.schemas import UserRegisterSchema
 from src.core.factories import UserRegisterSchemaFactory
 
-@pytest.mark.parametrize('password1, password2, result', [('testtest1', 'testtest1', does_not_raise()), ('testtest1', 'testtest2', pytest.raises(ValidationError))])
-def test_passwords_are_not_identical(password1, password2, result):
+@pytest.mark.parametrize(
+    "password1, password2, result",
+    [("testtest1", "testtest1", does_not_raise()), ("testtest1", "testtest2", pytest.raises(ValidationError))]
+)
+def test_user_register_schema_raises_validation_error_when_passwords_are_not_identical(password1, password2, result):
     with result:
         UserRegisterSchemaFactory.build(password=password1, password_repeat=password2)
         
-@pytest.mark.parametrize('future_date, result', [(datetime.now() - timedelta(days=1), does_not_raise()), (datetime.now() + timedelta(days=1), pytest.raises(ValidationError))])
-def test_date_is_from_future(future_date, result):
+@pytest.mark.parametrize(
+    "future_date, result",
+    [(datetime.now() - timedelta(days=1), does_not_raise()),
```

Comment:
nadal bym byl za formaterami kodu (isort, black), bo still zle sformatowane jest

Reply:
poza tym reszta spoko

Reply:
Tak wgl to czemu te rzeczy nie ida do mastera tylko do jakiegos brancha?

Reply:
w tej chwili tamten branch = master
wcześniej master był z tyłu z commitami i lepiej było porównać zmiany z tamtym branchem